tests/test_helpers.sh

```
@@ -0,0 +1,170 @@
+# Common functions used across all test suites
+
+# Colors (re-exported for test scripts)
+export RED='\033[0;31m'
+export GREEN='\033[0;32m'
+export YELLOW='\033[1;33m'
+export BLUE='\033[0;34m'
+export NC='\033[0m'
+
+# Print functions
+print_header() {
+    if [[ "$QUIET" != "true" ]]; then
```

Shellcheck (0.10.0)

[warning] QUIET is referenced but not assigned.

(SC2154)